packages/nest-commander/tsconfig.build.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "./../../tsconfig.build.json",
-  "include": ["src"]
+  "include": ["src"],
+  "compilerOptions": {
+    "moduleResolution": "Node16"
+  }
```

Comment:
To be clear for my knowledge, this is to make sure that the `await import` stays after compilation, yeah?

Reply:
Yep, just added a description to the pull request, accidentally opened it before adding one haha.